src/rgw/rgw_aio.h

```diff
@@ -27,6 +27,7 @@ namespace rgw {
 
 struct AioResult {
   rgw_raw_obj obj;
+  uint64_t id = 0; // id allows caller to associate a result with its request
```

Comment:
what will generate these?

Reply:
this is unused for writes in PutObjProcessor. for reads, `RGWRados::get_obj_iterate_cb()` generates them with:
```c++
const uint64_t id = obj_ofs; // use logical object offset for sorting replies
```
and the sorting happens in `get_obj_data::flush()`